apps/z2m_ikea_controller/z2m_ikea_controller.py

```diff
@@ -17,10 +17,10 @@
 
 
 def action(method):
-    def _action_impl(self, *args, **kwargs):
-        continue_call = self.before_action(method.__name__)
+    def _action_impl(self, *args):
+        continue_call = self.before_action(method.__name__, *args)
```

Comment:
Same would go for here, I would suggest to add the keyword arguments
```suggestion
continue_call = self.before_action(method.__name__, *args, **kwargs)
```